contracts/InterchainTokenService.sol

```diff
@@ -304,6 +304,11 @@ contract InterchainTokenService is
 
         if (deployer == interchainTokenFactory) {
             deployer = TOKEN_FACTORY_DEPLOYER;
+        } else if (bytes(destinationChain).length == 0) {
+            string memory destinationAddress = trustedAddress(chainName());
```

Comment:
```suggestion
            string memory destinationAddress = trustedAddressHash(chainName());
```
use `trustedAddressHash` instead, it's more efficient

Reply:
make sense, updated